espnet2/train/preprocessor.py

```diff
@@ -505,19 +505,24 @@ def __init__(
         self,
         train: bool,
         source_scp: str = None,
-        num_spk: int = 2,
+        num_utts: int = 2,
         dynamic_mixing_gain_db: float = 0.0,
         speech_name: str = "speech_mix",
         speech_ref_name_prefix: str = "speech_ref",
+        mixture_source_name: str = None,
         utt2spk: str = None,
     ):
 
         super().__init__(train)
         self.source_scp = source_scp
-        self.num_spk = num_spk
+        self.num_utts = num_utts
         self.dynamic_mixing_gain_db = dynamic_mixing_gain_db
         self.speech_name = speech_name
         self.speech_ref_name_prefix = speech_ref_name_prefix
+        if mixture_source_name is None:
+            self.mixture_source_name = f"{speech_ref_name_prefix}1"
+        else:
+            self.mixture_source_name = mixture_source_name
```

Comment:
Maybe add some comments to explain the meaning of this argument?

Reply:
good idea.